docs/manual/guide/production/ConductR.md

```diff
@@ -1,21 +1,43 @@
+ConductR is a container orchestration tool with the main goal of delivering operational productivity. ConductR is also designed to host your Lagom services with resilience.
 
-ConductR is free for development usage and there is a "sandbox" so that you can run ConductR locally and test your services. Please visit [ConductR's product page](http://lightbend.com/products/conductr) in order to download the sandbox, and also for more information on ConductR in general. If you'd like to know more about our commercial license then [please contact us](https://www.lightbend.com/company/contact). The remainder of this guide will discuss the specific integration points between Lagom and ConductR.
+ConductR is free for development usage and there is a "sandbox" so that you can run ConductR locally and test your services. For more information on ConductR go to:
+
+* [ConductR product page](http://lightbend.com/products/conductr)
+* [ConductR documentation](http://conductr.lightbend.com)
+
+This guide will explain how to:
+
+* Install ConductR sandbox
+* Package Lagom services
+* Start local ConductR cluster
+* Load and run services during and outside of development
+* Run Cassandra on ConductR
+ 
+If you'd like to know more about our commercial license then [please contact us](https://www.lightbend.com/company/contact).
+
+## Installing ConductR sandbox
+
+The ConductR sandbox is a docker image to easily create a ConductR cluster locally. To run ConductR on your development machine follow the instructions on the [ConductR developer page](https://www.lightbend.com/product/conductr/developer) up to the section **Configure Docker Machine**. We will run the sandbox later on in this guide. Note that in order to access this page you need to login with your Lightbend account.
 
 ## Packaging your services
 
-We have integrated the experience of packaging your Lagom services so that you can deliver them to ConductR with ease. By adding the [sbt-lagom-bundle plugin](https://github.com/typesafehub/sbt-lagom-bundle#lagom-bundle-plugin) you are able to package Lagom services for ConductR. In your project's `plugins.sbt`:
+We have integrated the experience of packaging your Lagom services so that you can deliver them to ConductR with ease. By adding the [sbt-lagom-bundle plugin](https://github.com/typesafehub/sbt-lagom-bundle#lagom-bundle-plugin) you are able to package Lagom services for ConductR. Add this plugin to your `project/plugins.sbt`:
 
 ```scala
 addSbtPlugin("com.typesafe.sbt" % "sbt-lagom-bundle" % "1.0.2")
 ```
 
-You can then package your services from within the activator console (we `reload` so that activator recognises the new plugin):
+Now, start the activator console from the terminal:
+
+```console
+$ activator
+```
+
+You can then package your services from within the activator console:
```

Comment:
Actually, the why is at the beginning of this section. I guess you can disregard this comment.